node_modules/oae-core/signin/css/signin.css

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#signin-modal {
+    text-align: center;
+}
+
+#signin-modal > .modal-dialog {
+    display: inline-block;
```

Comment:
It's set so that the modal just takes the width of its contents. there are lots of potential permutations of what content is inside, so this just ensures it is never excessively wide and never too skinny (to a maximum of 300px)